M/MathGL/build_tarballs.jl

```diff
@@ -3,7 +3,7 @@
 using BinaryBuilder, Pkg
 
 name = "MathGL"
-version = v"2.4.4"
+version = v"2.4.5"
```

Comment:
Didn't change the upstream source?

Reply:
Upstream is the same. Should we keep the same version number for the Julia package?

Reply:
@giordano, can we merge it?